dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1426,6 +1426,36 @@ def test_reductions_frame_dtypes_numeric_only(func):
     )
 
 
+@pytest.mark.parametrize("func", ["skew", "kurtosis"])
+def test_skew_kurt_numeric_only_false(func):
```

Comment:
Can we add an `assert_eq` to make sure the result from dask matches pandas?

 Also, is `numeric_only=True` tested elsewhere?

Reply:
Both of them are tested somewhere else, that's why I added only the False case here.